adafruit_esp32spi/adafruit_esp32spi_socket.py

```diff
@@ -193,6 +202,16 @@ def socknum(self):
         """The socket number"""
         return self._socknum
 
+    @property
+    def remote_ip(self):
+        """The remote ip"""
+        return self._remote_ip
+
+    @property
+    def remote_port(self):
+        """The remote port"""
+        return self._remote_port
+
```

Comment:
```suggestion
    def getpeername(self):
        ip, port = _the_interface.get_remote_data(self._socknum)
        # Parse so that ip is a string and port is an int
        # Second bullet here: https://docs.python.org/3/library/socket.html#socket-families
        return (ip, port)

```